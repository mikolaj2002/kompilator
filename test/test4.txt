# Test 4. Loops

PROGRAM IS
    x
IN
    x := 31;
    REPEAT
        WRITE x;
        x := x / 2;
    UNTIL x = 0;

    x := 31;
    WHILE x > 0 DO
        WRITE x;
        x := x / 2;
    ENDWHILE
END