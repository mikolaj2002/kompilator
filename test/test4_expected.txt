31
15
7
3
1
31
15
7
3
1
